include/zephyr/drivers/espi_emul.h

```diff
@@ -29,6 +24,11 @@
 extern "C" {
 #endif
 
+#include <zephyr/device.h>
```

Comment:
Moved inside the extern "C" clause to protect against https://stackoverflow.com/q/8526819

Reply:
I think this means that one of these files (or a file included from one of these) isn't using `extern "C"`